pre/remap_restart/remap_questions.py

```diff
@@ -13,383 +13,374 @@
-             720X410 \n \
-             1440X1080 \n ",
-            "choices": ['72X36','360X200','720X410','1440X1080'],
-            "when": lambda x: x['output:shared:model'] != 'data',
+            "message": "Select coupled ocean resolution for new restarts:\n",
+            "choices": choices_ogrid_cpld,
+            "when": lambda x: x['output:shared:omodel'] != 'data',
+        },
+
+        {
+            "type": "text",
+            "name": "output:air:nlevel",
+            "message": "Enter number of atmospheric levels for new restarts: (71 72 91 127 132 137 144 181)\n",
+            "default": "72",
         },
 
+        # to show the message, we ask output first
         {
             "type": "select",
-            "name": "input:shared:bc_base",
-            "message": "Select bcs base \n \
-             discover_ops: /discover/nobackup/projects/gmao/share/gmao_ops/fvInput/g5gcm/bcs \n \
-             discover_legacy: /discover/nobackup/projects/gmao/bcs_shared/legacy_bcs \n \
-             discover_couple: /discover/nobackup/projects/gmao/ssd/aogcm/atmosphere_bcs \n \
-             discover_ns: /discover/nobackup/projects/gmao/bcs_shared/fvInput/ExtData/esm/tiles \n \
-\n \
-Sample GCM tags for discover_ops, discover_legacy, discover_couple bc base :\n \
---------------- \n \
-G40   : Ganymed-4_0  .........  Heracles-5_4_p3 \n \
-ICA   : Icarus  ..............  Jason \n \
-GITOL : 10.3  ................  10.18 \n \
-INL   : Icarus-NL  ...........  Jason-NL \n \
-GITNL : 10.19  ...............  10.23 \n \
-\n \
-Sample DAS tags for discover_ops, discover_legacy, discover_couple bc base :\n \
---------------- \n \
-5B0  : GEOSadas-5_10_0_p2  ..  GEOSadas-5_11_0 \n \
-512  : GEOSadas-5_12_2  .....  GEOSadas-5_16_5\n \
-517  : GEOSadas-5_17_0  .....  GEOSadas-5_24_0_p1\n \
-525  : GEOSadas-5_25_1  .....  GEOSadas-5_29_4\n  \
-\n \
-Sample BC version for discover_ns ( new structure BC base): \n \
--------------- \n \
-NL3   : Newland version 3 \n \
-NL4   : Newland version 4 \n \
-NL5   : Newland version 5 \n \
-v06   : Not generated yet \n",
-            "choices": ["discover_ops", "discover_legacy", "discover_couple","discover_ns", "other"],
-            "when": lambda x: not x['input:shared:MERRA-2'],
+            "name": "input:shared:bc_version",
+            "message": f'''\nSelect boundary conditions (BCs) version of input restarts:
+    BCs version      | ADAS tags            | GCM tags typically used with BCs version
+    -----------------|----------------------|-----------------------------------------
+    GM4: Ganymed-4_0 | 5_12_2 ... 5_16_5    | Ganymed-4_0      ... Heracles-5_4_p3
+    ICA: Icarus      | 5_17_0 ... 5_24_0_p1 | Icarus, Jason    ... 10.18   
+    NL3: Icarus-NLv3 | 5_25_1 ... present   | Icarus_NL, 10.19 ... present
+    ----------------------------------------------------------------------------------
+    Other: Additional choices used in model or DAS development.
+              \n\n ''',
```

Comment:
Same comment as above.  Can this multi-line message (which occurs twice) be put into a variable?